### ISSUE TITLE
Some fontLigatures config not work in latest version `Maple Mono NF CN`

### ISSUE THREAD

**Issue by chenweiyi** [subframe7536/maple-font#200]
### Question
When I use the latest version(V7.0 Beta20) `Maple Mono NF CN` ,  some fontLigatures not work.

`===`, `!==`, `>=` and so on not work. But some charactor work , eg: '=>'

As a comparison, the `Maple Mono SC NF` that I used before can work.

### Env
mac: 11.7.4 intel core i5
vscode: 1.90.0

### Additional Information
My vscode config :
```
"editor.fontFamily": "'Maple Mono NF CN','Maple Mono SC NF'",
"editor.fontLigatures": "'calt', 'liga', 'dlig', 'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss06', 'ss07', 'ss08'",
```

**Comment by subframe7536** [comment]
see doc in [V7](https://github.com/subframe7536/maple-font/tree/variable?tab=readme-ov-file#features)

tldr: the recommend config is

```
"editor.fontFamily": "'Maple Mono NF CN'",
"editor.fontLigatures": "'calt'",
```
